geometry_msgs/Twist velocity
float32 duration